[IndexFilters] Prevent keyboard shortcuts working if component has no search field or filters (#11665)
Shopify/polaris: .changeset/sour-walls-kneel.md

---
'@shopify/polaris': patch
---

> Updated IndexFilters to not add the keyboard shortcuts when there is no search field or filters

Reviewer: ```suggestion
Fixed `IndexFilters` responding to keyboard shortcuts when there is no search field or filters
```
